djangocms_text_ckeditor/static/djangocms_text_ckeditor/ckeditor_plugins/cmswidget/plugin.js

```diff
@@ -164,7 +164,7 @@
                     var contents = $(this.element.$).children();
                     var displayProp = contents.css('display');
 
-                    if (displayProp !== 'inline' && displayProp !== 'inline-block') {
+                    if (!displayProp.includes('inline')) {
```

Comment:
Uncaught TypeError: Cannot read properties of undefined (reading 'includes')

Reply:
It seems `$.css` can return `undefined`. Will fix. 👍